Lib/logging/__init__.py

```diff
@@ -252,6 +251,18 @@ def _releaseLock():
         _lock.release()
 
 
+def _get_lock() -> ContextManager:
```

Comment:
```suggestion
def _get_lock():
```

We don't use type annotations within the stdlib.

A

Reply:
Ah good to know! Thank you